### ISSUE TITLE
Allow "overwrite" logic to be passed over

### ISSUE THREAD

**Issue by ablwr** [jronallo/video-sprites#7]
Right now, a user will be prompted if there's a filename that already exists, but it might be good to add a flag that ignores that particular flow so the gem can be wrapped up as a microservice that doesn't care and doesn't want to be prompted ever

**Comment by nzoss** [comment]
Yeah, I noted that in the change – that or a ‘force’ that overwrites, for non-interactive uses?

**Comment by jronallo** [comment]
This sounds like a good change @ablwr and whatever name that makes sense to you is fine by me.
